### PR TITLE
feat: support for graphemes in key input

### PR DESCRIPTION
**Pull request by sadym-chromium**
Allow graphemes in key input according to step 6 of https://w3c.github.io/webdriver/#dfn-process-a-key-action.
Use [`Intl.Segmenter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter) to detect grapheme borders.

Follow up: update WPT tests.

### CODE REVIEW

File: src/bidiMapper/modules/input/ActionDispatcher.ts [hunk 1]
```diff
@@ -878,3 +902,8 @@ function getRadii(
     radiusY: height ? height / 2 : 0.5,
   };
 }
+
+function splitGraphemes(value: string) {
```
Comment (OrKoN) [inline src/bidiMapper/modules/input/ActionDispatcher.ts:906]: let's add some unit tests for the grapheme detection and split

File: src/bidiMapper/modules/input/ActionDispatcher.ts [hunk 2]
```diff
@@ -878,3 +902,8 @@ function getRadii(
...
+  const segmenter = new Intl.Segmenter(undefined, {granularity: 'grapheme'});
```
Comment (OrKoN) [inline src/bidiMapper/modules/input/ActionDispatcher.ts:907]: Is it dependent on the current locale? would not it lead to incorrect results and differences between node vs browser?

File: src/bidiMapper/modules/input/ActionDispatcher.ts (hunk 1 above)
Comment (sadym-chromium) [reply src/bidiMapper/modules/input/ActionDispatcher.ts:906]: good point. Added.

File: src/bidiMapper/modules/input/ActionDispatcher.ts (hunk 2 above)
Comment (sadym-chromium) [reply src/bidiMapper/modules/input/ActionDispatcher.ts:907]: According to https://www.unicode.org/reports/tr29/#Grapheme_Cluster_Boundaries, there can be some edge-cases in different locales. So I stick to `en` for consistency.

Comment (OrKoN) [reply src/bidiMapper/modules/input/ActionDispatcher.ts:907]: I see that undefined is passed to Intl.Segmenter and the current locale is not guaranteed to be en_US. Can you explicitly set it here?

File: src/bidiMapper/modules/input/ActionDispatcher.ts (hunk 1 above)
Comment (OrKoN) [reply src/bidiMapper/modules/input/ActionDispatcher.ts:906]: Not seeing the tests yet

File: src/bidiMapper/modules/input/ActionDispatcher.ts (hunk 2 above)
Comment (sadym-chromium) [reply src/bidiMapper/modules/input/ActionDispatcher.ts:907]: haven't pushed yet

**Review by OrKoN — APPROVED** [review]

File: src/bidiMapper/modules/input/ActionDispatcher.ts (hunk 2 above)
Comment (sadym-chromium) [reply src/bidiMapper/modules/input/ActionDispatcher.ts]: pushed now

**Pull request merged**